tests/performance_tests/log_reader_tests.py

```diff
@@ -3,13 +3,15 @@
 # Also ensures that all versions of nodeos logs can be handled
 import log_reader
 
+from pathlib import Path
+
 testSuccessful = False
 
 # Test log scraping for 3.2 log format
 dataCurrent = log_reader.chainData()
 dataCurrent.startBlock = None
 dataCurrent.ceaseBlock = None
-log_reader.scrapeLog(dataCurrent, "tests/performance_tests/nodeos_log_3_2.txt.gz")
+log_reader.scrapeLog(dataCurrent, Path("tests")/Path("performance_tests")/Path("nodeos_log_3_2.txt.gz"))
```

Comment:
The `/` operator supported by `Path` accepts strings as well as `Path` objects, including chains of strings, as long as the first part is a `Path`.  I prefer to write this sort of thing as `Path('tests') / 'performance_tests' / 'nodeos_log_3_2.txt.gz'` since it's less verbose.  This includes f-strings.